src/libstore/s3_binary_cache_store.go

```diff
@@ -0,0 +1,101 @@
+package libstore
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"net/url"
+	"github.com/minio/minio-go/v7"
+	"github.com/minio/minio-go/v7/pkg/credentials"
+	"log"
+)
+
+// S3BinaryCacheStore ...
+type S3Store struct {
+	Profile string // The name of the AWS configuration profile to use.
+	Region string // "aws-region"
```

Comment:
Where is the region attribute being used in the code?

Reply:
when we return in the `NewStoreReader`